src/index.js

```diff
@@ -17,7 +17,11 @@ import users from './json/users'
 let initialState = {
     categoriesKey: categories,
     productsKey: products,
-    usersKey: users
+    usersKey: users,
+    cartKey: [
```

Comment:
Sign me up for a new car.

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/1068aaa3-b8a5-4d43-9891-ebeb6ae23994-7d8d6038-9387-48b0-82be-4ba3176f9fe9-v1.png?transparent=1&palette=1&width=246)